src/ex1/script.js

```diff
@@ -0,0 +1,82 @@
+let pendingTaskCount = 0;
+
+const taskInput = document.getElementById("todoTaskInput");
+const clearButton = document.getElementById("clearButtonId");
+
+taskInput.addEventListener("keypress", function(event) {
+    if (event.key === "Enter") {
+        event.preventDefault();
+        document.getElementById("addTaskButtonId").click();
+    }});
+
+if(pendingTaskCount === 0){
+    startingPInsert();
+    displyPendingTasks();
+}
+
+function startingPInsert(){
+    const startingP = document.createElement("p");
+    startingP.classList.add("startingP");
+    startingP.innerHTML = "Start To Arrange Your Tasks!";
+    const divTodoTasksListContainer = document.querySelector("div.todoTasksListContainer");
+    divTodoTasksListContainer.insertBefore(startingP, divTodoTasksListContainer.firstChild);
+    clearButton.disabled = true;
+}
+
+function onAddTaskButton(){
+    if(taskInput.value.length === 0){
+        alert("Your trying to enter a task empty");
+    }
+    else{
+        clearButton.disabled = false;
+        const startingP = document.querySelector("p.startingP");
+        if(pendingTaskCount === 0){
+            startingP.remove();
+        }
+        const taskList = document.querySelector("ul");
+        let liTag = document.createElement("li");
```

Comment:
naming naming naming naming 🤔 liTag doesn't mean anything to anyone else who'll read this code other than you. think about your teammates